src/org/thoughtcrime/securesms/ConversationItem.java

```diff
@@ -305,6 +307,36 @@ protected void setInteractionState(DcMsg messageRecord, boolean pulseHighlight)
     }
   }
 
+  private void setContentDescription(DcMsg messageRecord) {
```

Comment:
```suggestion
  private void setContentDescription() {
```

parameter is not used